src/functionalTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/UpdateCaseTest.java

```diff
@@ -21,8 +21,26 @@
 
     @Test
     public void should_update_case() throws JSONException {
+        updateCaseAndVerifyResponse(
+            "updatecase/valid-request.json",
+            "updatecase/valid-response.json"
+        );
+    }
+
+    @Test
+    public void should_update_case_by_exception_record_with_auto_update_fields_without_warnings() throws JSONException {
```

Comment:
What do you mean by `without_warnings`?

Reply:
Will remove `without_warnings` to avoid confusion